File: Data Structures & Algorithms/concatenation-of-array/submission-5.java
class Solution {
    public int[] getConcatenation(int[] nums) {
        int n= nums.length;
        int ans[] = new int[2*n];
        for(int i=0;i<n;i++)
        {
            ans[i] = nums[i];
            if(i>=0 && i<n)
            ans[i+n] =nums[i];
        }
        return ans;
    }
}

File: Data Structures & Algorithms/two-integer-sum/submission-5.py
class Solution:
    def twoSum(self, nums: List[int], target: int) -> List[int]:
        d = {}
        for i in range(len(nums)):
            val = target - nums[i]
            if val in d:
                return ([d[val],i])
            d[nums[i]] = i
        